### ISSUE TITLE
index.es5.min.js:2 Uncaught TypeError: Cannot set property 'Countries' of undefined

### ISSUE THREAD

**Issue by vgavro** [annexare/Countries#22]
Got this error on es5.min.js load (v2.3.0). Should I load some dependencies before? And if yes - anyway it would be more useful If you could load this without external dependencies.
```
index.es5.min.js:2 Uncaught TypeError: Cannot set property 'Countries' of undefined
```

**Comment by vgavro** [comment]
https://jsfiddle.net/muy9h50d/2/ link to check out (open console)

**Comment by z-ax** [comment]
Thanks, will check out the build.

**Comment by z-ax** [comment]
Published as `v2.3.1`.
